### DISCUSSION TITLE
Offcloud links inside zip files dont play

### DISCUSSION THREAD

**Discussion by MrXT21** [mhdzumair/MediaFusion#122] [Q&A]
I have noticed that alot of links pulled by torrentio and mediafusion for offcloud are inside zip files for the entire season however none of these links play. They appear to play fine for both RD and AD. Im not sure if theres an issue with the links that are being scrapped as if its pulling a direct link to the zip file rather than the media file contained within. If i go directly to offcloud and click explore on the zip file the media inside is of course a different link. Is this fixable or is this an offcloud issue?

**Comment by mhdzumair** [comment]
continue from https://github.com/mhdzumair/MediaFusion/issues/123
